tinytroupe/environment/tiny_world.py

```
@@ -616,6 +643,22 @@ def _pretty_step(self, cur_step, total_steps, timedelta_per_step=None):
 
         return rendering
 
+    def _pretty_intervention(self, intervention):
+        indent = "          > "
+        justification = textwrap.fill(
+            intervention.precondition_justification(),
+            width=TinyPerson.PP_TEXT_WIDTH,
```

Ruff (0.8.2)

`TinyPerson` may be undefined, or defined from star imports

(F405)